pkg/service/controller.go

```
@@ -228,8 +230,32 @@ func (c *ControllerService) ListSnapshots(context.Context, *csi.ListSnapshotsReq
 }
 
 //ControllerExpandVolume
-func (c *ControllerService) ControllerExpandVolume(context.Context, *csi.ControllerExpandVolumeRequest) (*csi.ControllerExpandVolumeResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "")
+func (c *ControllerService) ControllerExpandVolume(_ context.Context, req *csi.ControllerExpandVolumeRequest) (*csi.ControllerExpandVolumeResponse, error) {
+	klog.Infof("Expanding volume %v to %v bytes.", req.VolumeId, req.CapacityRange.RequiredBytes)
+
+	conn, err := c.ovirtClient.GetConnection()
+	if err != nil {
+                msg := fmt.Sprintf("Failed to get ovirt client connection")
+		klog.Errorf(msg)
+		return nil, status.Error(codes.Unavailable, msg)
+	}
+
+	// find diskAttachment and diskAttachmentService by DiskId
+	diskAttachment, diskAttachmentService, _ := diskAttachmentByDisk(conn, req.VolumeId)
```

Why ignore the error?